code/modules/vehicles/mecha/combat/durand.dm

```diff
@@ -158,20 +158,20 @@ own integrity back to max. Shield is automatically dropped if we run out of powe
 	light_power = 5
 	light_color = LIGHT_COLOR_ELECTRIC_CYAN
 	light_on = FALSE
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF //The shield should not take damage from fire,  lava, or acid; that's the mech's job.
 	///Our link back to the durand
 	var/obj/vehicle/sealed/mecha/combat/durand/chassis
 	///To keep track of things during the animation
 	var/switching = FALSE
-	var/currentuser
-	resistance_flags = LAVA_PROOF | FIRE_PROOF | ACID_PROOF //The shield should not take damage from fire,  lava, or acid; that's the mech's job.
-
 
-/obj/durand_shield/Initialize(mapload, _chassis, _layer, _dir)
+/obj/durand_shield/Initialize(mapload, _chassis, _plane, _layer, _dir)
```

Comment:
since you're touching this, do you mind updating this to proper style guidelines?
```
/obj/durand_shield/Initialize(mapload, chassis, plane, layer, dir)
    src.chassis = chassis
    src.layer = layer
    ...
```